nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -45,6 +45,7 @@ def get_char_dataset(config: dict, augmentor: Optional['AudioAugmentor'] = None)
         normalize=config.get('normalize_transcripts', False),
         trim=config.get('trim_silence', False),
         parser=config.get('parser', 'en'),
+        manifest_line_parser=config.get('manifest_line_parser', None),
```

Comment:
A Callable function can't really be part of a manifest - so how is this handled ?

Reply:
add hydra.utils.instantiate()

Reply:
Previously it was a string method embedded in NeMo yes? So now its a user modifiable hydra block?

Reply:
yes, now it is hydra block